crml/cennzx-spot/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Parity Technologies (UK) Ltd. and Centrality Investments Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Runtime API definition required by CENNZX RPC extensions.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::{Codec, Decode, Encode};
+use sp_runtime::{traits::SimpleArithmetic, RuntimeDebug};
+
+/// A result of querying the exchange
+#[derive(Eq, PartialEq, Encode, Decode, RuntimeDebug)]
+pub enum CennzxSpotResult<Balance> {
```

Comment:
Using Result is the standard way of handling the results of functions/methods. It would be also consistent with our code anywhere else.